apps/api/prisma/seed/index.js

```diff
@@ -24,9 +24,58 @@ async function main() {
 }
 
 main().then(() => {
-  console.log('done')
+  console.log('done');
+  console.log('running fetchAPISAVE');
+   fetchAPISaveGenes();
 }).catch(e => {
   console.error(e)
 }).finally(async () => {
   await prisma.$disconnect()
 })
+
+// const { PrismaClient } = require('@prisma/client');
+// const prisma = new PrismaClient();
+
+async function fetchAPI() {
```

Comment:
Need a better name, maybe `fetchHgncData`

Reply:
updated function name to fetchHgncData

Reply:
Did you push your changes?